Bowling/Bowling/FrameNoBonusRollFactory.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class FrameNoBonusRollFactory
+    {
+        public const int totalPinsCount = 10;
+        public int PlaceCountHolder { get; set; }
+        
+        public Frame Create(int frameNumber, string bowlingMarks, int placeCount)
+        {
+            PlaceCountHolder = placeCount;
+            return ScoreRolls(frameNumber, bowlingMarks);
+        }
+
+        private Frame ScoreRolls(int frameNumber, string bowlingMarks)
```

Comment:
I would encourage you to try and find ways to make this function shorter.  I think an easy way to do it is to make 2 functions, one for each of your `switch` statements.

Reply:
I shortened this up, created some additional functions and removed all the switch code.